pkg/micropython/doc.txt

```diff
@@ -0,0 +1,74 @@
+/**
+ * @defgroup pkg_micropython MicroPython RIOT port
+ * @ingroup  pkg
+ * @brief    MicroPython - Python for microcontrollers
+ *
+ * # MicroPython RIOT package
+ *
+ * "MicroPython is a lean and efficient implementation of the Python 3
+ *  programming language that includes a small subset of the Python standard
+ *  library and is optimised to run on microcontrollers and in constrained
+ *  environments."
+ *
+ * @see https://micropython.org
+ *
+ * ## Status
+ *
+ * MicroPython on RIOT has to be considered experimental. While the basic
+ * interpreter works fairly well on native and Cortex-M, it has not seen much
+ * testing.
+ *
+ * ## Implementation details
+ *
+ * The RIOT port of MicroPython currently resides in a fork at
+ * https://github.com/kaspar030/micropython (in branch add_riot_port). It is
+ * based on Micropython's "ports/minimal" with some extra modules enabled.
+ * It re-uses the gc_collect code from ports/unix, which has special support
+ * for i386 and Cortex-M. On other platforms, it uses setjmp() to collect
+ * registers.
+ *
+ * ## MicroPython's test suite
+ *
+ * It is possible to run MicroPython's test suite for testing this port.
+ *
+ * Steps:
+ *
+ * 1. flash tests/pkg_micropython
+ * 2. cd tests/pkg_micropython/bin/pkg/${BOARD}/micropython
+ * 3. git apply ports/riot/slow_uart_writes.patch
+ * 4. cd tests
+ * 5. ./run-tests --target pyboard --device ${PORT}
+ *
+ * ## MicroPython modules
+ *
+ * Currently, the port enables only a subset of the available MycroPython
+ * modules. See "ports/riot/mpconfigport.h" for details.
+ *
+ * For now, the utime module has RIOT specific code and should work as expected.
+ *
+ * ## RIOT specific modules
+ *
+ * Currently, these are implemented:
+ *
+ * ### thread_getpid()
+ *
+ *     $ import riot
+ *     $ print(riot.thread_getpid())
+
+ * ### xtimer
+ *
+ *     $ import xtimer
```

Comment:
I would replace the prompt character with the micropython one, e.g replace `$` with `>>>`. Same above for `thread_getpid()` example.

Reply:
done